### DISCUSSION TITLE
tiktok的直播能下载吗

### DISCUSSION THREAD

**Discussion by dizhu108878989** [Johnserf-Seed/TikTokDownload#615] [Q&A]
我测试国内抖音正常可下载的，但是国外的tiktok就有点抓瞎了

**Comment by Johnserf-Seed** [comment]
最新的 `F2` 版本已支持，同时已支持douyin 与 tiktok直播弹幕获取。
![image](https://github.com/user-attachments/assets/7c151999-ccbb-4549-a3ad-d6b5789e78c6)
